ocs_ci/utility/aws.py

```diff
@@ -1094,6 +1094,73 @@ def get_stack_name_by_instance_id(self, instance_id):
 
         return stack_name
 
+    def get_hosted_zone_id(self, cluster_name):
+        """
+        Get Zone if from given cluster_name
```

Comment:
Just some styling issue... Change this line to "Get zone id from given cluster_name"